draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1333,13 +1335,14 @@ than the previously generated UUID. If this is not the case then any number
 of things could have occurred, such as clock rollbacks,
 leap second handling, and counter rollovers. Applications SHOULD embed sufficient
 logic to catch these scenarios and correct the problem to ensure that the next
-UUID generated is greater than the previous. To handle this scenario, the
+UUID generated is greater than the previous, or at least report an appropriate error.
+To handle this scenario, the
 general guidance is that application MAY reuse the previous timestamp and
 increment the previous counter method.
 
 ## UUID Generator States {#generator_states}
 
-The UUID generator state only needs to be read from stable storage once at boot
+The (optional) UUID generator state only needs to be read from stable storage once at boot
```

Comment:
I know we say MAY in the next paragraph but when I read this again it seemed to communicate that this was an expected part of the implementation, so I think adding this helps.